ui-v2/package.json

```diff
@@ -40,6 +46,7 @@
     "ember-cli-app-version": "^3.0.0",
     "ember-cli-autoprefixer": "^0.8.1",
     "ember-cli-babel": "^6.6.0",
+    "ember-cli-cjs-transform": "^1.2.0",
```

Comment:
Hmmm not sure, think this can go now actually, it's probably left over from when I was testing out `broccolli` bits, will check and see if its removable, good spot!

Reply:
Nah, you defo need it, just tried it out. It's weird though because it's actually needed by a dependency, I remember trying to get it to work by specifying it in the dependency instead, but as far as I remember that didn't work either.

I've put this in my backlog to relook at it at a later date